experimental/web/index.html

```diff
@@ -6,8 +6,10 @@
   </head>
 
   <body>
+    <script src="https://raw.githack.com/protobufjs/protobuf.js/v6.10.1/dist/protobuf.js"></script>
```

Comment:
I've used this unofficial protobuf lib before for the node.js example before, and while it's convenient due dynamically reading proto files, I found it to be somewhat buggy. 

For instance it sometimes omits number fields with the value `0`. (Common JS pitfall, where people check whether a field exists via checking for truthyness instead of undefinedness.

Reply:
Good to know, yes I only used this because it can parse proto files at runtime, but I expect a proper version of this app would just pre-compile them from sources anyways.